R/series__series.R

```diff
@@ -265,7 +265,7 @@ Series_struct = method_as_active_binding(\() series_make_sub_ns(self, expr_struc
 #' s2 = pl$Series(1:3, "a", dtype = pl$Float32)
 #' s2
 pl_Series = function(
-    x,
+    values,
```

Comment:
The problem here is that code that used `x = foo` will be silently captured by `...`, which breaks the expected usage. Can you check that `x` is in `...` and throw a warning/error if it is the case? (and add a test for this scenario)

Reply:
In my opinion, if such usage were done, an error would occur somewhere in the code and people would notice the breaking change.
Of course we can capture `x` as you suggest, but I don't think it is worth doing as I have written here.
https://github.com/pola-rs/r-polars/pull/939#pullrequestreview-1946651267